packages/framework-provider-kubernetes/package.json

```diff
@@ -27,7 +27,9 @@
     "form-data": "^3.0.0",
     "isomorphic-fetch": "^2.2.1",
     "node-fetch": "^2.6.1",
-    "redis": "^3.0.2"
+    "redis": "^3.0.2",
+    "mustache": "4.1.0",
+    "axios": "0.21.1"
```

Comment:
You didn't modify the `framework-provider-kubernetes` package. Why do you need this dependencies?

Reply:
We were adding those dependencies manually in the user project, and now we are adding them in the package